File: 250216/첫 번째와 두 번째 교환/exchange-1st-and-2nd.cpp
#include <string>
#include <iostream>
using namespace std;

int main() {
    string str;

    cin >> str;
    char first, second;

    first = str[0];
    second = str[1];
    
    for (int i = 0; i < str.length(); i++) {
        if (str[i] == first)
            str[i] = second;
        else if (str[i] == second)
            str[i] = first;
    }
    cout << str << endl;
    return 0;
}

File: 250216/첫 번째와 두 번째 교환/exchange-1st-and-2nd.js
const fs = require("fs");

const str = fs.readFileSync(0, "utf8").trim();

const [first, second, ...rest] = str;

const result = [second, first, ...rest.map(ch => {
    if (ch == first)
        return (second);
    else if (ch == second)
        return (first);
    return (ch);
}
)].join("");

console.log(result);
